src/trace.c

```diff
@@ -992,6 +992,34 @@ fix_path (char *pathname)
 }
 
 
+/* Look for given pathname from the hash table,
+ * and insert new one if it's not in it yet.
+ * 
```

Comment:
Small nit, but this line ends with an extra space character.
Shows up in git diff or git show.

Can you remove it and submit again?

Reply:
Done.